test/metabase/db/schema_migrations_test.clj

```diff
@@ -1430,3 +1431,50 @@
           :date_joined      :%now})
         (migrate!)
         (is (false? (sample-content-created?)))))))
+
+(deftest cache-config-migration-test
+  (testing "Caching config is correctly copied over"
+    (impl/test-migrations "v50.2024-04-12T12:33:09" [migrate!]
+      (mdb.query/update-or-insert! :model/Setting {:key "enable-query-caching"} (constantly {:value "true"}))
+      (mdb.query/update-or-insert! :model/Setting {:key "query-caching-ttl-ratio"} (constantly {:value "100"}))
+      (mdb.query/update-or-insert! :model/Setting {:key "query-caching-min-ttl"} (constantly {:value "123"}))
+
+      (let [db   (t2/insert-returning-pk! Database (-> (mt/with-temp-defaults Database)
```

Comment:
```suggestion
      (let [db   (t2/insert-returning-pk! :metabase_database (-> (mt/with-temp-defaults Database)
```

Generally we should avoid triggering hooks for migration tests as it often includes future behavior. One example is that we might introduce a new column where hook will add a default value, in that case this test will fail because the new column does not exist.

Reply:
done